src/component/TooltipBoundingBox.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { CSSProperties, PureComponent, ReactNode } from 'react';
+import { translateStyle } from 'react-smooth';
+import { AllowInDimension, AnimationDuration, AnimationTiming, CartesianViewBox, Coordinate } from '../util/types';
+import { getTooltipTranslate } from '../util/tooltip/translate';
+
+export type TooltipBoundingBoxProps = {
+  active: boolean;
+  allowEscapeViewBox: AllowInDimension;
+  animationDuration: AnimationDuration;
+  animationEasing: AnimationTiming;
+  children: ReactNode;
+  coordinate: Partial<Coordinate>;
+  hasPayload: boolean;
+  isAnimationActive: boolean;
+  offset: number;
+  position: Partial<Coordinate>;
+  reverseDirection: AllowInDimension;
+  useTranslate3d: boolean;
+  viewBox: CartesianViewBox;
+  wrapperStyle: CSSProperties;
+};
+
+type State = {
+  dismissed: boolean;
+  dismissedAtCoordinate: Coordinate;
+};
+
+const EPS = 1;
```

Comment:
Nit: If you know what EPS stands for, I would sure love to replace this with a proper name.

Reply:
"Epsilon"

> denotes the smallest quantity like a term which is taken as the zero in some limit

Reply:
This is no longer necessary. It seems to have been added due to the structure of obtaining the tooltip's viewbox, which could have caused an infinite loop